microsetta_interface/templates/kits.jinja2

```diff
@@ -23,6 +23,11 @@
         .tooltip.bs-tooltip-top .tooltip-arrow::before {
             border-top-color: #0000ff !important;
         }
+        .new_kit_sample_container {
```

Comment:
Can you please move these changes into the [appropriate CSS class](https://github.com/biocore/microsetta-interface/blob/master/microsetta_interface/static/css/minimal_interface.css#L1210) in `static/css/minimal_interface.css`?